sway/main.c

```diff
@@ -251,6 +251,18 @@ static void drop_permissions(bool keep_caps) {
 #endif
 }
 
+void enable_debug_flag(const char *flag) {
+	if (strcmp(flag, "render-tree") == 0) {
+		enable_debug_tree = true;
+	} else if (strncmp(flag, "damage=", 7) == 0) {
+		damage_debug = &flag[7];
+	} else if (strcmp(flag, "txn-debug")) {
+		txn_debug = true;
+	} else if (strncmp(flag, "txn-timeout", 11) == 0) {
```

Comment:
Missing `=`?

Reply:
Yeah, not sure why GitHub has marked this as outdated. This is still relevant; everything else looks okay to me.

Reply:
Cause it's outdated, I totally fixed that

Reply:
I would expect the fix to be:

    } else if (strncmp(flag, "txn-timeout=", 12) == 0) {
    		txn_timeout_ms = atoi(&flag[12]);

ie. s/txn-timeout/txn-timeout=/ and replacing 11 with 12.

Reply:
oh derp I thought this comment had been referring to two lines up where I forgot to check the return value of strcmp for txn-debug